editor/src/components/inspector/inspector-common.tsx

```diff
@@ -61,23 +60,42 @@ function stringToFlexDirection(str: string | null): FlexDirection | null {
   }
 }
 
-export function detectFlexAlignJustifyContent(
+const DefaultJustifyContentFlexAlign: [FlexJustifyContent, FlexAlignment] = [
+  'flex-start',
+  'flex-start',
+]
+
+function detectFlexAlignJustifyContentOne(
   metadata: ElementInstanceMetadataMap,
   elementPath: ElementPath,
 ): [FlexJustifyContent, FlexAlignment] {
```

Comment:
I must have overlooked or missed this before, and I don't want to block this PR on something unrelated, but please can we replace this return value with an object

Reply:
done! (https://github.com/concrete-utopia/utopia/pull/2982/commits/39a3f3293b12d49d8beedcc159aaf27c091a025b)